test/environments-create-delete.spec.ts

```diff
@@ -29,4 +30,14 @@ describe('Create and delete environments', () => {
 
     await tests.spectron.client.waitForExist('.header input[placeholder="No environment"]');
   });
+
+  it('Add ten environments ever clicking in the first', async () => {
+    let port = 3000;
+    for (let i = 0; i < 10; i++) {
```

Comment:
You could reduce the code a little bit and directly use `port` as counter variable
```suggestion
    for (let port = 3000; port < 3010; port++) {
```